cmd/tailscale/cli/ping.go

```diff
@@ -50,6 +50,7 @@ relay node.
 		fs.BoolVar(&pingArgs.verbose, "verbose", false, "verbose output")
 		fs.BoolVar(&pingArgs.untilDirect, "until-direct", true, "stop once a direct path is established")
 		fs.BoolVar(&pingArgs.tsmp, "tsmp", false, "do a TSMP-level ping (through IP + wireguard, but not involving host OS stack)")
+		fs.BoolVar(&pingArgs.icmp, "icmp", false, "do a ICMP-level ping (through IP + wireguard. Remote host OS stack only)")
```

Comment:
the "Remote host OS stack only" makes sense to me but only because I've thought about this a lot.

Most users will be confused. I see it kinda matches the line above, but will they sort correctly? it might be confusing in alphabetical order.

Maybe: "send an ICMP-level ping via wireguard (bypasses local host network stack)"

Reply:
I went back and forward with some different language. I think where I landed is ok, trying to keep it terse and relatively broadly understandable.